application/migrations/0048_auto_20201116_1410.py

```diff
@@ -0,0 +1,95 @@
+                ('hero_background_image_mobile_fi', models.ForeignKey(blank=True, null=True, on_delete=django.db.models.deletion.SET_NULL, related_name='+', to='application.CustomImage', verbose_name='Pääkuva mobiililla FI')),
+                ('hero_background_image_mobile_sv', models.ForeignKey(blank=True, null=True, on_delete=django.db.models.deletion.SET_NULL, related_name='+', to='application.CustomImage', verbose_name='Pääkuva mobiililla SV')),
+                ('hero_background_image_sv', models.ForeignKey(blank=True, null=True, on_delete=django.db.models.deletion.SET_NULL, related_name='+', to='application.CustomImage', verbose_name='Pääkuva SV')),
+                ('hero_top_layer_image_en', models.ForeignKey(blank=True, null=True, on_delete=django.db.models.deletion.SET_NULL, related_name='+', to='application.CustomImage', verbose_name='Pääkuvan päälle asettuva kuva EN')),
+                ('hero_top_layer_image_fi', models.ForeignKey(blank=True, null=True, on_delete=django.db.models.deletion.SET_NULL, related_name='+', to='application.CustomImage', verbose_name='Pääkuvan päälle asettuva kuva FI')),
+                ('hero_top_layer_image_sv', models.ForeignKey(blank=True, null=True, on_delete=django.db.models.deletion.SET_NULL, related_name='+', to='application.CustomImage', verbose_name='Pääkuvan päälle asettuva kuva SV')),
+                ('social_media_image_en', models.ForeignKey(blank=True, null=True, on_delete=django.db.models.deletion.SET_NULL, related_name='+', to='application.CustomImage', verbose_name='Some-postauksen kuva EN')),
+                ('social_media_image_fi', models.ForeignKey(blank=True, null=True, on_delete=django.db.models.deletion.SET_NULL, related_name='+', to='application.CustomImage', verbose_name='Some-postauksen kuva FI')),
+                ('social_media_image_sv', models.ForeignKey(blank=True, null=True, on_delete=django.db.models.deletion.SET_NULL, related_name='+', to='application.CustomImage', verbose_name='Some-postauksen kuva SV')),
+            ],
+            options={
+                'verbose_name': 'Banner',
+            },
+            bases=('wagtailcore.page',),
+        ),
+        migrations.AddField(
+            model_name='landingpages',
+            name='bottom_banner',
+            field=models.ForeignKey(blank=True, null=True, on_delete=django.db.models.deletion.SET_NULL, related_name='+', to='application.BannerPages', verbose_name='Bottom banner'),
+        ),
+        migrations.AddField(
+            model_name='landingpages',
+            name='top_banner',
+            field=models.ForeignKey(blank=True, null=True, on_delete=django.db.models.deletion.SET_NULL, related_name='+', to='application.BannerPages', verbose_name='Top banner'),
+        ),
+        migrations.RunPython(migrate_banner_data, migrations.RunPython.noop)
```

Comment:
Apart from this line:

```python
```

and `migrate_banner_data` function, the rest of this file is automatically generated by Django right?

Reply:
Yes, only the migrate_banner_data migrate function is manually created, the rest was generated by Django